Assignment-2-Crud(updated).c

```diff
@@ -0,0 +1,231 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+typedef struct User
+{
+    int Id;
+    char Name[50];
+    int Age;
+} User;
+
+void addUser();
+void displayUsers();
+void updateUser();
+void deleteUser();
+int checkUserExists(int id); 
+User getInputUserDetails();  
+void printUserDetails(User user); 
+
+int main()
+{
+    int choice;
+
+    while (1)
+    {
+        printf("\n__________________ User CRUD Operation ______________________\n\n");
+        printf("1. Add New User\n2. Display All Users\n3. Update User\n4. Delete User\n5. Exit\n");
+        printf("\nEnter your choice: ");
+        scanf("%d", &choice);
+
+        switch (choice)
+        {
+        case 1:
+            addUser();

```

Comment:
after trying to add user in same user ID and when displayed list , it is adding junk data to file , can you please take care of this?